megamek/src/megamek/common/weapons/AreaEffectHelper.java

```diff
@@ -394,29 +399,19 @@ public static void artilleryDamageEntity(Entity entity, int damage, Building bld
             return;
```

Comment:
Up about 30 LOC there seems a possible null dereferencing in regards to `hex` as it's checked somewhere within 100 LOC below this line. Right after the comment about `//check: is entity inside building?`

Reply:
Always edge cases can derail us so good to be safe. :)